backend/dal/Repositories/AcquisitionFileRepository.cs

```diff
@@ -103,6 +107,8 @@ public PimsAcquisitionFile Add(PimsAcquisitionFile acquisitionFile)
                 }
             }
 
+            // TODO: Review this logic in the context of an addition
+            // - there should be no pre-existing AcquisitionFilePerson rows for this file!
             foreach (var acquisitionPeople in acquisitionFile.PimsAcquisitionFilePeople)
             {
                 if (acquisitionPeople.AcquisitionFilePersonId != 0)
```

Comment:
Added another TODO to be reviewed by @Maheshbabu28  and @devinleighsmith 
Not sure if I'm missing something here but we don't need to worry about existing rows in the ACQ -> Person intermediate table when creating a new ACQ file, do we?

Reply:
Yes, this logic doesn't make sense, there would never be pre-existing rows. Low priority but should be changed.